### PR TITLE
migrate code from gitlab

### PR DESCRIPTION
**Pull request by engelmi**
Code migrated from: https://gitlab.com/autosd-on-yocto/meta-bluechi

/cc @lrossetti

### CODE REVIEW

File: COPYING.MIT
Comment (mwperina) [inline COPYING.MIT:1]: Could you please rename the file to LICENSE, so it alings with GH projects standard?
And would it be possible to use MIT-0 license as described in https://spdx.org/licenses/ ?

File: README
Comment (mwperina) [inline README:1]: Could you please rename to README.md, so it's more clear that the file has markdown syntax and not plain text?

Comment (engelmi) [reply README:1]: Done :heavy_check_mark:

File: COPYING.MIT
Comment (engelmi) [reply COPYING.MIT:1]: Done :heavy_check_mark:

File: recipes-core/bluechi/files/bluechi-agent.conf
Comment (mwperina) [inline recipes-core/bluechi/files/bluechi-agent.conf:1]: Could you please copy the content of the file from https://github.com/eclipse-bluechi/bluechi/blob/main/config/agent/agent.conf so it's aligned with bluechi-controller.conf structure?

Also is NodeName enough? Shouldn't we set also ManagerHost?

Comment (engelmi) [reply recipes-core/bluechi/files/bluechi-agent.conf:1]: Done :heavy_check_mark: 
Yes, setting only NodeName should be fine. ManagerHost will default to localhost, which is what we want for a single-node setup as far as I understood.

**Review by mwperina — APPROVED** [review]
LGTM

File: LICENSE [hunk 1]
```diff
@@ -0,0 +1,18 @@
+The MIT-Zero License
+
+Copyright (c) <year> <copyright holders>
```
Comment (odra) [inline LICENSE:3]: I think we need to assign values to both year and holder in here?

Comment (mwperina) [reply LICENSE:3]: Good point, we missed that even in BlueChi. So how about `Copyright (c) 2023 Red Hat`?

Comment (mwperina) [reply LICENSE:3]: Or we can use more upstream one `Copyright (c) 2023 BlueChi Authors`?

Comment (odra) [reply LICENSE]: I think "Copyright (c) 2023 Red Hat" makes more sense.

**Review by mwperina — APPROVED** [review]
LGTM

File: LICENSE (hunk 1 above)
Comment (engelmi) [reply LICENSE]: Changed to `Copyright (c) 2023, Red Hat, Inc.`.

**Pull request merged**